### ISSUE TITLE
Fatal error: sleef.h: No such file or directory

### ISSUE THREAD

**Issue by EikanWang** [pytorch/pytorch#87546]
### 🐛 Describe the bug

Could not find `sleef.h` when I tried to use `at::vec` to vectorize the code as follows.

[vec256_float.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/cpu/vec/vec256/vec256_float.h#L10) includes `sleef.h` if it is to be compiled with `CPU_CAPABILITY_AVX2` defined. But `sleef.h` is not installed in the release package. So it cannot pass build if the system also does not install `sleef` library.

C++ source file: test.cpp
```C++
#include <ATen/cpu/vec/functional.h>
#include <ATen/cpu/vec/vec.h>

extern "C" void kernel(const float* __restrict__ in_ptr0,
                       const float* __restrict__ in_ptr1,
                       float* __restrict__ out_ptr0)
{
    #pragma omp parallel num_threads(48)
    {
        #pragma omp for
        for(long i0=0; i0<8; i0+=1)
        {
            auto tmp0 = at::vec::Vectorized<float>::loadu(in_ptr0 + 8*i0);
            auto tmp3 = at::vec::Vectorized<float>::loadu(in_ptr1 + 8*i0);
            auto tmp1 = tmp0.abs();
            auto tmp2 = tmp1.sin();
            auto tmp4 = tmp2 + tmp3;
            tmp4.store(out_ptr0 + 8*i0);
        }
        #pragma omp for simd simdlen(4)
        for(long i0=64; i0<70; i0+=1)
        {
            auto tmp0 = in_ptr0[i0];
            auto tmp3 = in_ptr1[i0];
            auto tmp1 = std::abs(tmp0);
            auto tmp2 = std::sin(tmp1);
            auto tmp4 = tmp2 + tmp3;
            out_ptr0[i0] = tmp4;
        }
    }
}
```

Build Command:
```bash
g++ -shared -fPIC -Wall -std=c++14 -Wno-unused-variable -I/opt/conda/lib/python3.7/site-packages/torch/include -I/opt/conda/lib/python3.7/site-packages/torch/include/torch/csrc/api/include -I/opt/conda/lib/python3.7/site-packages/torch/include/TH -I/opt/conda/lib/python3.7/site-packages/torch/include/THC -I/opt/conda/include/python3.7m -L/opt/conda/lib/python3.7/site-packages/torch/lib -L/opt/conda/lib -lc10 -ltorch -ltorch_cpu -ltorch_python -lgomp -DCPU_CAPABILITY_AVX2 -march=native -O3 -ffast-math -fno-finite-math-only -fopenmp -o test.so test.cpp
```

### Versions

[pip3] numpy==1.23.3
[pip3] torch==1.12.1
[pip3] torchaudio==0.12.1
[pip3] torchvision==0.13.1
[conda] blas                      1.0                         mkl
[conda] cpuonly                   2.0                           0    pytorch
[conda] ffmpeg                    4.3                  hf484d3e_0    pytorch
[conda] mkl                       2021.4.0           h06a4308_640
[conda] mkl-service               2.4.0            py38h7f8727e_0
[conda] mkl_fft                   1.3.1            py38hd3c417c_0
[conda] mkl_random                1.2.2            py38h51133e4_0
[conda] numpy                     1.22.3                   pypi_0    pypi
[conda] numpy-base                1.23.3           py38h31eccc5_0
[conda] pytorch                   1.12.1              py3.8_cpu_0    pytorch
[conda] pytorch-mutex             1.0                         cpu    pytorch
[conda] torch                     1.11.0+cpu               pypi_0    pypi
[conda] torchaudio                0.11.0+cpu               pypi_0    pypi
[conda] torchvision               0.12.0+cpu               pypi_0    pypi

cc @malfet @seemethere @zou3519

**Comment by EikanWang** [comment]
@malfet , I think we should add `sleef.h` in the release package to fix the issue as the user could use pytorch as the library mode. any comments?

**Comment by EikanWang** [comment]
@malfet , any comments? We are using `at::vec` to vectorize the kernel explicitly for TorchInductor. Hence, it is critical for TorchInductor CPU performance.

**Comment by EikanWang** [comment]
The issue has been fixed in #87068.
